src/libs/auth.ts

```diff
@@ -55,16 +55,62 @@ const getAuthInstance = () => {
   return authStore.get().authContext;
 };
 
-export const signOut = () => {
-  // TODO: invalidate runtime cookies https://broadworkbench.atlassian.net/browse/IA-3498
+export const enum SignOutCauses {
+  requested,
+  disabled,
+  declinedTos,
+  expiredRefreshToken,
+  errorRefreshingAuthToken,
+  idleStatusMonitor,
+  unspecified,
+}
+
+const sendSignOutMetrics = (causes: SignOutCauses): void => {
```

Comment:
```suggestion
const sendSignOutMetrics = (cause: SignOutCauses): void => {
```

Nit: the cause is singular. `causes` suggests that this function takes a list of causes or something like that.